src/styles/components/_component-card.scss

```
@@ -73,7 +73,10 @@
   }
```

Here's the suggestion for adding a focus state to `.component-link`

```suggestion
  }
  
  &:focus {
    @media screen and (prefers-color-scheme: dark) {
      background-color: black;
    }
  }
```

---

Thanks for pairing with me to review how to better use the inspector to visualize these. I'm happy I was able to find that class that controls the default background color for the grid! :)